Sources/UIKit/TextAlignmentConstraint.swift

```diff
@@ -0,0 +1,162 @@
+//
+//  TextAlignmentConstraint.swift
+//
+//  Created by Cameron Pulsford on 10/4/16.
+//  Copyright © 2016 Raizlabs. All rights reserved.
+//
+
+#if os(OSX)
+    import AppKit
+#else
+    import UIKit
+#endif
+
+@objc(BONTextAlignmentConstraintAttribute)
+public enum TextAlignmentConstraintAttribute: Int {
```

Comment:
Would prefer to have this nested inside `TextAlignmentConstraint`, so it's `BonMot.TextAlignmentConstraint.Attribute`. The Obj-C name is good, though.